doc/dev/developer_guide/dash-devel.rst

```diff
@@ -1026,6 +1026,9 @@ Alerts and notifications
 Default notification should have `text-info` color. Success notification should
 have `text-success` color. Failure notification should have `text-danger` color.
 
+:ref:`Accessibility <accessibility>`
+....................................
+
```

Comment:
@nSedrickm this needs to be removed since you put "Accessibility" in a new section.